clippy_lints/src/wildcard_imports.rs

```diff
@@ -143,7 +143,7 @@ impl LateLintPass<'_> for WildcardImports {
                 } else {
                     // In this case, the `use_path.span` ends right before the `::*`, so we need to
                     // extend it up to the `*`. Since it is hard to find the `*` in weird
-                    // formattings like `use _ ::  *;`, we extend it up to, but not including the
+                    // forms like `use _ ::  *;`, we extend it up to, but not including the
```

Comment:
happy to drop if people are really attached to it.

Reply:
'formattings' makes it clear that it's about how the thing is formatted. The awkward spacing in example being the only weird thing _should_ make that apparent, but it's possible to not pick up on that right away.

Given that the only benefit here is being six characters shorter, it's better to revert this.